src/electionguard/election.py

```diff
@@ -773,6 +773,19 @@ def _generate_contests_with_placeholders(
         return contests
 
 
+@dataclass(frozen=True)
+class ElectionConstants(Serializable):
+    """
+    The constants for mathematical functions during the election. 
+    """
```

Comment:
I really like this. An interesting thing to ponder is rummaging around the rest of the codebase to make sure everybody uses these parameters rather than the global constants. That would be really beneficial to support "test mode" parameters. (https://github.com/microsoft/electionguard-python/issues/76)